web/frontend/src/components/slam-2d-render.vue

```diff
@@ -411,7 +410,7 @@ watch(() => props.pointCloudUpdateCount, () => {
 <template>
   <div
     ref="container"
-    class="pcd-container relative w-full"
```

Comment:
this has been removed since `pcd-container` has not visual effect on the display of the 2D map.